dkron/store.go

```diff
@@ -149,6 +149,14 @@ func (s *Store) SetJob(job *Job, copyDependentJobs bool) error {
 		if err := s.removeFromParent(ej); err != nil {
 			return err
 		}
+
+		if ej == nil || ej.ParentJob == "" {
```

Comment:
ej can't be nil at this point, no need to check this

Reply:
yes, you are right. I have clean this.